doc/content/reference/cli/troubleshooting.md

```diff
@@ -7,10 +7,53 @@ This section contains help for common errors encountered when using the CLI.
 
 <!--more-->
 
+## Faulty OAuth Server Address
+
+In the [CLI configuration file]({{< ref "/getting-started/cli/installing-cli#configuration" >}}), the server address is set to `thethings.example.com` by default. If you do not change this server address to the address of your own deployment, using `ttn-lw-cli login` command will redirect to `https://thethings.example.com/oauth/authorize?client_id=cli&redirect_uri=local-callback&response_type=code` in your web browser, where the CLI needs to be authorized for exchanging an access token.
+
+Since your server address is most likely different than `thethings.example.com`, you will see an error about that URL not being found in your browser, the access token will not be obtained and you will not be able to execute any further CLI commands. In your terminal, you will see something like:
+
+```
+INFO Opening your browser on https://thethings.example.com/oauth/authorize?client_id=cli&redirect_uri=local-callback&response_type=code
+INFO After logging in and authorizing the CLI, we'll get an access token for future commands.
+INFO Waiting for your authorization...
+```
+
+The solution to this issue would be to replace `thethings.example.com` occurrences with the server address of your {{% tts %}} deployment.
+
+{{< note >}} You will experience the same behavior if you are using an `https` port other than `443` and you have not specified that port in the `oauth-server-address` parameter in the CLI configuration file (or with `--oauth-server-address` flag when running the CLI). For example, if you are running {{% tts %}} on `localhost`, you would have to specify the OAuth server address as `'https://localhost:8885/oauth'`.
+{{</ note >}}
+
+## Certificate Signed by Unknown Authority
+
+If the CA file path is not specified in the [CLI configuration file]({{< ref "/getting-started/cli/installing-cli#configuration" >}}) (or with `--ca` flag when running the CLI), or if you are using self-signed certificates, using `ttn-lw-cli login` might result in the following error:
+
+```
+ERROR Could not exchange OAuth access token    error=Post "https://thethings.example.com:8885/oauth/token": x509: certificate signed by unknown authority
+```
+
+In this case, the CLI configuration file should contain `ca: /path/to/ca.pem`, or you should run the CLI with `--ca="/path/to/ca.pem"` flag. If using self-signed certificates, you should also import the CA file using your web browser's certificate manager.
```

Comment:
Since we suggest that people use the `ttn-lw-cli use $hostname` command, we could also have a note here for the `--fetch-ca` flag, which helps in the case of self-signed certificates.

The only reason _against_ mentioning this flag is that people should only be using self-signed certificates for development environments and not actual deployments, so leaving this up to you.